frontend/src/concepts/pipelines/topology/usePipelineTaskTopology.tsx

```diff
@@ -106,41 +125,40 @@ const getNodesForTasks = (
     const pipelineTask: PipelineTask = {
       type: 'groupTask',
       name: taskName,
-      steps: executor ? [executor.container] : undefined,
+      steps: executor?.container ? [executor.container] : undefined,
```

Comment:
This is a one-liner fix for [RHOAIENG-6572](https://issues.redhat.com/browse/RHOAIENG-6572)